client/package.json

```diff
@@ -227,7 +227,6 @@
 	"devDependencies": {
 		"@automattic/calypso-eslint-overrides": "workspace:^",
 		"@sentry/webpack-plugin": "^1.21.0",
-		"@storybook/addon-actions": "^7.6.20",
```

Comment:
Removing some of these explicit dependencies, as they are already included through the `@automattic/calypso-storybook` config.

(Same with the other package.jsons)